models/DataObject/ClassDefinition/Data.php

```diff
@@ -738,6 +738,13 @@ public function getGetterCodeObjectbrick($brickClass)
         $code .= '*/' . "\n";
         $code .= 'public function get' . ucfirst($key) . " () {\n";
 
+        if ($this instanceof DataObject\ClassDefinition\Data\Relations\AbstractRelations && $this->getLazyLoading()) {
```

Comment:
@weisswurstkanone we should move this part into `preGetData()`, this makes the migration script obsolete